docs/api/components/schemas.yaml

```
@@ -1524,7 +1524,7 @@ Compact-status:
       description: |-
         **Applicable to attachment compaction only**
 
-        This is the amount of attachments that have been marked to be purged.
+        This is the amount of attachments that have been marked to be kept.
```

Still not really accurate

```suggestion
        This is the number of references there are to legacy attachments. 
```